components/error-summary/src/index.tsx

```diff
@@ -123,7 +123,7 @@ const ErrorSummary = ({ onHandleErrorClick, heading, description, errors, ...pro
       <UnorderedList mb={0} listStyleType="none">
         {errors.map((error, index) => (
           <ListItem key={error.targetName}>
-            <StyledErrorText tabIndex={index + 1} onClick={() => onHandleErrorClick(error.targetName)}>
+            <StyledErrorText tabIndex={index + 1} onClick={() => onHandleErrorClick?.(error.targetName)}>
```

Comment:
seems this is our first use of the optional chaining operator inside a component (we've used it elsewhere inside some stories and examples)

the ancient version of react-docgen we had been using didn't like this, which broke CI - fortunately it looks like the latest release works OK

out of curiosity I tried removing the hack we have for ensuring components get documented to see if that issue is resolved with the latest react-docgen - no joy

Reply:
some more details on that issue here https://github.com/govuk-react/govuk-react/issues/81#issuecomment-841740534